.github/workflows/docker.yaml

```diff
@@ -143,12 +143,15 @@ jobs:
         with:
           context: .
           load: true
-          tags: inventree/inventree:test
+          tags: inventree-test
 
       - name: Test Docker Image
         id: test-docker
         run: |
-          docker run --rm inventree/inventree:test invoke --list
+          docker run --rm inventree-test invoke --version
+          docker run --rm inventree-test invoke --list
+          docker run --rm inventree-test gunicorn --version
+          docker run --rm inventree-test pg_dump --version
```

Comment:
maybe we later could also run a small integration test with the docker container before pushing. Maybe just run inventree and try to curl the api to see if django started correctly?

Reply:
I was considering this, we would need to spin up a database container also. Might be worth using a MySQL container in this instance for testing